modyn/evaluator/internal/metric_manager.py

```diff
@@ -0,0 +1,13 @@
+from modyn.evaluator.internal.metrics import AbstractEvaluationMetric
+from modyn.utils import dynamic_module_import
+
+
+class MetricManager:
```

Comment:
sorry, the purpose of this class not yet clear to me by its name. So does this allow us to instantiate a metric by its name? Maybe it is a `MetricFactory` then with a function `from_name` that returns an AbstractEvaluationMetric?

Reply:
(in that case I think from_name can also be static)